### PR TITLE
Refs #23919 -- Removed encoding preambles and future imports

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: django/db/backends/mysql/base.py [hunk 1]
```diff
@@ -4,8 +4,6 @@
 Requires mysqlclient: https://pypi.python.org/pypi/mysqlclient/
 MySQLdb is supported for Python 2 only: http://sourceforge.net/projects/mysql-python
```
Comment (matthewwillian) [inline django/db/backends/mysql/base.py]: Stale docstring?

Comment (claudep) [inline django/db/backends/mysql/base.py]: Sure, but this patch targeted specifics removal. Further patches will clean the code by grepping for `python 2`.

**Comment by duaneking** [comment]
So to be clear for people reading this, the intention is not to support Python 2 at all in the Django v3 release?

**Comment by charettes** [comment]
@duaneking it's actually Django 2.0, which should be released in a few weeks, that doesn't support Python 2 anymore; Django 1.11 LTS is the latest version of Django to support Python 2 but it will be supported until 2020 which coincides with the EOL of Python 2.

**Pull request merged**